{"name": "test_main[cases48]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/home/chenlei/backup_sqls/apiautotest_2022-01-06T14_21_46.sql'", "trace": "@pytest.fixture(scope=\"session\")\n    def data_clearing():\n        \"\"\"数据清洗\"\"\"\n        DataClearing.server_init()\n        # 1. 备份数据库\n>       DataClearing.backup_mysql()\n\ntest\\conftest.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntools\\data_clearing.py:119: in backup_mysql\n    cls.server.files_action(0,\ntools\\data_clearing.py:77: in files_action\n    self.ftp_client.get(remotepath=remote_path, localpath=file_path)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <paramiko.sftp_client.SFTPClient object at 0x0000017F68EF2460>\nremotepath = '/root/backup_sql/apiautotest_2022-01-06T14_21_46.sql'\nlocalpath = '/home/chenlei/backup_sqls/apiautotest_2022-01-06T14_21_46.sql'\ncallback = None\n\n    def get(self, remotepath, localpath, callback=None):\n        \"\"\"\n        Copy a remote file (``remotepath``) from the SFTP server to the local\n        host as ``localpath``.  Any exception raised by operations will be\n        passed through.  This method is primarily provided as a convenience.\n    \n        :param str remotepath: the remote file to copy\n        :param str localpath: the destination path on the local host\n        :param callable callback:\n            optional callback function (form: ``func(int, int)``) that accepts\n            the bytes transferred so far and the total bytes to be transferred\n    \n        .. versionadded:: 1.4\n        .. versionchanged:: 1.7.4\n            Added the ``callback`` param\n        \"\"\"\n>       with open(localpath, \"wb\") as fl:\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/chenlei/backup_sqls/apiautotest_2022-01-06T14_21_46.sql'\n\nD:\\PythonCode\\apiAutoTest\\lib\\site-packages\\paramiko\\sftp_client.py:801: FileNotFoundError"}, "parameters": [{"name": "cases", "value": "['c49', '服务质量列表查询', '{\"Authorization\": \"${token}\"}', 'config/qos/list', 'get', 'params', '', '', '{\"configType\":\"$.data.list.0.configType\", \"key\":\"$.data.list.0.key\"}', '', '{\"$.code\": 0}']"}], "start": 1641450328727, "stop": 1641450328727, "uuid": "2bb8f4c2-b944-4c4d-b1f5-5e350b3e90b3", "historyId": "d90cba7385e9471943c57cd1332d0ac5", "testCaseId": "881516e7ef2e2aa5436f6a3c3e6a5bcb", "fullName": "test.test_api#test_main", "labels": [{"name": "tag", "value": "flaky(reruns=2, reruns_delay=1)"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "DESKTOP-BNVFIE4"}, {"name": "thread", "value": "23664-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_api"}]}